oidc-fapi-conformance-tests/run-local.sh

```
@@ -0,0 +1,486 @@
+
+  if command -v nginx >/dev/null 2>&1; then
+    info "nginx already installed"
+  else
+    STARTED_NGINX="yes"
+    if [[ "$(uname -s)" == "Darwin" ]]; then
+      command -v brew >/dev/null 2>&1 || die "nginx not found and Homebrew unavailable to install it"
+      info "Installing nginx via Homebrew..."
+      brew install nginx || die "brew install nginx failed"
+    else
+      info "Installing nginx via apt..."
+      sudo apt update && sudo apt install -y nginx || die "apt install nginx failed"
```

Shellcheck (0.11.0)

[info] Note that A && B || C is not if-then-else. C may run when A is true.

(SC2015)